core-services/ci-operator-configresolver/ci-operator-configresolver.yaml

```diff
@@ -53,11 +57,16 @@ items:
     name: ci-operator-configresolver
     namespace: ci
   spec:
+    host: ci-operator-configresolver-ci.svc.ci.openshift.org
     port:
       targetPort: main
     to:
       kind: Service
       name: ci-operator-configresolver
+      weight: 100
+  status:
```

Comment:
Do we really want to update `status` for each of `routers`?

Reply:
What if we set `status` to `{}`?

Reply:
```bash
oc --as=system:serviceaccount:ci:config-updater apply -f core-services/ci-operator-configresolver/ci-operator-configresolver.yaml --validate --dry-run
error: error validating "core-services/ci-operator-configresolver/ci-operator-configresolver.yaml": error validating data: ValidationError(Route.status): missing required field "ingress" in com.github.openshift.api.route.v1.RouteStatus; if you choose to ignore these errors, turn validation off with --validate=false
```

Reply:
OK. Puzzel solved.

status became optional last month:
https://github.com/openshift/api/blame/master/route/v1/types.go#L36

same for host
https://github.com/openshift/api/blame/master/route/v1/types.go#L71

They all got fixed by the same commit:
https://github.com/openshift/api/commit/52ffecf235bf379c0adff33eb6011f01eb1a785d

@stevekuznetsov and @petr-muller 
Let me know if you want me to update routes now, or wait until we upgrade our sever with the commit above.

Reply:
Some hints @deads2k gave me on this topic:
https://coreos.slack.com/archives/CB48XQ4KZ/p1579185643154900